### ISSUE TITLE
Player "mistake" + "puzzle finished" sound effect

### ISSUE THREAD

**Issue by Sreenika-dev** [eternaleclipse/chess-mazes#61]
Would like to add another sound for the dialog box that shows up when hanging piece is attacked (like wrong buzzer sound)
Reference to the sounds : https://youtu.be/AHLCE7iH9qI

**Comment by eternaleclipse** [comment]
Hey, sounds awesome (no pun intended 🙂)
Since the SFX video you've linked does not explicitly specify a license, I think we can use something from FreeSound with Creative Commons Zero (CC0) license.

Searching for "wrong", I found a couple of candidates:
https://freesound.org/people/Raclure/sounds/483598/
https://freesound.org/people/AbdrTar/sounds/558121/
https://freesound.org/people/unadamlar/sounds/476177/
https://freesound.org/people/TheBuilder15/sounds/415764/

The site just became unavailable, but I'm assuming there are good SFX for "correct" too, that can be added when a puzzle is finished.

If you want to work on this, I can assign you this issue. Thanks!

**Comment by Sreenika-dev** [comment]
Yes please assign it to me, i will work on it by getting a Creative Commons Sound effect

**Comment by eternaleclipse** [comment]
The PR was closed automatically as a result of the recent merge we made.
https://github.com/chess-mazes/chess-mazes/pull/70

I'm working on merging your changes to the new codebase and they are appreciated!
